.snippets/code/builders/ethereum/precompiles/utility/nonspecific/signMessage.js

```diff
@@ -0,0 +1,20 @@
+const { Web3 } = require('web3');
+const web3 = new Web3('https://rpc.api.moonbase.moonbeam.network');
+
+// Address and Private Key
+const address = '0xCD8Bcd9A793a7381b3C66C763c3f463f70De4e12';
+const pk1 =
+  '0x3f6e4d3bd1f8f465c8f1c8cb32fd79d123a6deda7965a1f7bc36bc79b5abe162';
```

Comment:
Please remove the PK here and use filler like INSERT_ALICES_PRIVATE_KEY. 
You can also replace the address above to either ALICE's address or INSERT_ALICE_ ADDRESS